bin/configs/go-array-default-value.yaml

```diff
@@ -0,0 +1,5 @@
+generatorName: go
+outputDir: samples/client/others/go/array-default-value
```

Comment:
thanks for the pr

please add the new folder to the go github action wrokflow so that it will be tested moving forward: https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-go-client.yaml